images/02-console/docker-init

```diff
@@ -11,10 +11,15 @@ done
 
 DOCKER_BIN=$(which docker) || DOCKER_BIN=/usr/bin/docker
 
-for i in /opt/bin /usr/local/bin; do
+for i in /opt/bin /usr/local/bin /var/lib/rancher/docker; do
+    if [ -x ${i}/dockerd ]; then
```

Comment:
This is the biggest part that I was concerned about. Since the binaries are now supplied via the Docker container, there aren't any binaries in the console container until they're copied. I think most users expect them to be in `/usr/bin` even if they aren't directly launching the Docker daemon.

Another important piece of this is the client. We'd have to update the PATH in the console itself for the client to be located in `/var/lib/rancher/docker`.